app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count_mask = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            count_mask += 1
+    if count_mask > 0:
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/219699987-73c5df1d-8266-4ee8-9903-ef330dea2fd2.png)
fix this, please